### ISSUE TITLE
npm run start błąd

### ISSUE THREAD

**Issue by NataD** [daftcode/frontend_levelup_2018#17]
Nie mogę użyć npm run start, za każdym razem dostaję błąd:

![image](https://user-images.githubusercontent.com/20286494/39670548-81d9846a-5107-11e8-8774-4c3d3f2be3eb.png)

Nic nie zmieniałam w plikach daftcode-react-starter.

**Comment by michalwiacek** [comment]
Hej
> Nic nie zmieniałam w plikach daftcode-react-starter.
To znaczy, że sklonowalaś repo daftcode-react-starter i próbujesz odpalić?

**Comment by NataD** [comment]
Tak. Teraz już działa. Sklonowała jeszcze raz i zadziałało wszystko...
